trymito.io/pages/excel-to-python/functions/math/ABS.tsx

```diff
@@ -0,0 +1,353 @@
+                      </tr>
+                      <tr>
+                        <td>=ABS(2*-2)</td>
+                        <td>Calculate the absolute value of 2 * -2</td>
+                        <td>4</td>
+                      </tr>
+                    </tbody>
+                  </table>
+              </section>
+
+              {/* Equivalent Python Code Using Pandas */}
+              <section className={excelToPythonStyles.section}>
+                  <h2 
+                    id="Implementing ABS in Pandas"
+                    className={excelToPythonStyles.link}
+                  >
+                    Implementing the Absolute Value function in Pandas
+                    <a className={excelToPythonStyles.section_copy} href="#Implementing ABS in Pandas">#</a>
+                  </h2>
+                  <p>
+                    To replicate the ABS function in Excel using Python and Pandas, you can use the `abs()` function available in Pandas. Below are examples of how to achieve the same functionality.
+                  </p>
+                  
+                  <h3 
+                    id="ABS of every cell" 
+                    className={classNames(excelToPythonStyles.h3, excelToPythonStyles.link)}
+                  >
+                    Calculate the absolute value of every cell in a Pandas series
+                    <a className={excelToPythonStyles.section_copy} href="#ABS of every cell">#</a>
+                  </h3>
+                  <p>
+                    The most common way to use the function in Excel is to apply it directly to a column or series of numbers in a Pandas DataFrame.
+                  </p>
+
+                  <CodeBlock code={`# Calculate the absolute value of the Numbers column
+df['ABS_Result'] = df['Numbers'].abs()`}
+                  />
+                  <h3 
+                    id="ABS difference" 
+                    className={classNames(excelToPythonStyles.h3, excelToPythonStyles.link)}
+                  >
+                    Finding the absolute difference between two columns
+                    <a className={excelToPythonStyles.section_copy} href="#ABS difference">#</a>
+                  </h3>
+                  <p>
+                    To use the absolute value as part of a more complex operation, you can use the `apply()` function to apply the operation to every element in an pandas dataframe column.
+                  </p>
+                  <CodeBlock code = {`# Calculate the absolute difference between Column1 and Column2
+df['Absolute_Difference'] = (df['Column1'] - df['Column2']).abs()`
+                  }/>
+                  <h3 
+                    id="Complex ABS operations" 
+                    className={classNames(excelToPythonStyles.h3, excelToPythonStyles.link)}
+                  >
+                    Using ABS as part of a more complex operation
+                    <a className={excelToPythonStyles.section_copy} href="#Complex ABS operations">#</a>
```

Comment:
Clicking on these href's scrolls super weird - farther than you want -- on safari.

Reply:
I'm not sure exactly how to fix this one. So leaving for now.